components/gitpod-db/src/workspace-db.spec.db.ts

```diff
@@ -307,13 +307,6 @@ class WorkspaceDBSpec {
         expect(dbResult.length).to.eq(0);
     }
```

Comment:
not super satisfied with removing the only test we have for this method, but since it is implicitly tested by the relationship-updater a bit and only used there (atm) I'm can live without it for now.